### PR TITLE
Added Exception Handling entry to `document/core/appendix/changes.rst`

### PR DESCRIPTION
**Pull request by ioannad**
Please note:

Apart from try-catch and try-delegate, all other instructions in `changes.rst` are given without the immediates or any arguments that follow them. Since try-catch and try-delegate are the only new structured block instructions I guessed it is ok to include the full form of these new instructions here.

### CODE REVIEW

**Review by aheejin — APPROVED** [review]
Thank you!

File: document/core/appendix/changes.rst [hunk 1]
```diff
@@ -138,6 +138,24 @@ Added vector type and instructions that manipulate multiple numeric values in pa
 * New injection/projection :ref:`vector instructions <syntax-instr-vec>`: :math:`\K{i}\!N\!\K{x}\!M\!\K{.splat}`, :math:`\K{f}\!N\!\K{x}\!M\!\K{.splat}`, :math:`\K{i}\!N\!\K{x}\!M\!\K{.bitmask}`
 
 
+.. index:: instructions, exception, tag type, tag, handler
+
+Exception Handling
+..................
+
+Added tag type, instructions that throw exceptions of a tag type, and instructions that handle exceptions. [#proposal-eh]_
```
Comment (rossberg) [inline document/core/appendix/changes.rst]:
```suggestion
Added tag definitions, imports, and exports, and instructions to throw and catch exceptions [#proposal-eh]_
```

File: document/core/appendix/changes.rst [hunk 2]
```diff
@@ -138,6 +138,24 @@ Added vector type and instructions that manipulate multiple numeric values in pa
...
+
+* New :ref:`tag type <syntax-tagtype>`: :math:`[t^\ast]\to[]`-
```
Comment (rossberg) [inline document/core/appendix/changes.rst]: I'd replace this with tag definitions, imports, and exports. Tag types are more of an auxiliary notion to support these constructs.

File: document/core/appendix/changes.rst [hunk 3]
```diff
@@ -138,6 +138,24 @@ Added vector type and instructions that manipulate multiple numeric values in pa
...
+
+* New :ref:`tag section <binary-tagsec>` in binary format.
```
Comment (rossberg) [inline document/core/appendix/changes.rst]: Perhaps move this to the end of the list.

File: document/core/appendix/changes.rst [hunk 4]
```diff
@@ -138,6 +138,24 @@ Added vector type and instructions that manipulate multiple numeric values in pa
...
+
+* New exception throwing :ref:`control instructions <syntax-instr-control>`: :math:`\THROW` and :math:`\RETHROW`.
+
+* New handler :ref:`control instructions <syntax-instr-control>`: :math:`\TRY~\X{bt}~\instr_1^\ast~(\CATCH~x~\instr_2^\ast)^\ast~(\CATCHALL~\instr_3^\ast)^?\END`, :math:`\TRY~\X{bt}~\instr^\ast~\DELEGATE~l`.
```
Comment (rossberg) [inline document/core/appendix/changes.rst]:
```suggestion
* New handler :ref:`control instructions <syntax-instr-control>`: :math:`(\TRY~\X{bt}~\instr_1^\ast~(\CATCH~x~\instr_2^\ast)^\ast~(\CATCHALL~\instr_3^\ast)^?\END)` and :math:`(\TRY~\X{bt}~\instr^\ast~\DELEGATE~l)`.
```

File: document/core/appendix/changes.rst [hunk 5]
```diff
@@ -138,6 +138,24 @@ Added vector type and instructions that manipulate multiple numeric values in pa
...
+
+* New uncaught exception :ref:`result <syntax-result>`.
```
Comment (rossberg) [inline document/core/appendix/changes.rst]: I'd remove this, it's more like an implication, not a feature.

File: document/core/appendix/changes.rst [hunk 6]
```diff
@@ -155,3 +173,6 @@ Added vector type and instructions that manipulate multiple numeric values in pa
 
 .. [#proposal-vectype]
    https://github.com/WebAssembly/spec/tree/main/proposals/simd/
+
+.. [#proposal-eh]
```
Comment (rossberg) [inline document/core/appendix/changes.rst]:
```suggestion
.. [#proposal-exceptions]
```

**Review by rossberg — APPROVED** [review]

**Pull request merged**